src/commands/System/Admin/cdn.ts

```diff
@@ -0,0 +1,48 @@
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { SkyraCommand } from '#lib/structures';
+import { PermissionLevels } from '#lib/types/Enums';
+import { ApplyOptions } from '@sapphire/decorators';
+import { fetch, FetchResultTypes } from '@sapphire/fetch';
+import { send } from '@sapphire/plugin-editable-commands';
+import { Message, MessageAttachment } from 'discord.js';
+
+@ApplyOptions<SkyraCommand.Options>({
+	description: LanguageKeys.Commands.System.CdnDescription,
+	detailedDescription: LanguageKeys.Commands.System.CdnExtended,
+	permissionLevel: PermissionLevels.BotOwner,
+	subCommands: ['get', 'upload', 'delete']
+})
+export class UserCommand extends SkyraCommand {
+	public async get(message: Message, args: SkyraCommand.Args) {
+		const name = await args.pick('string');
+
+		const { cdn } = this.container.grpc;
+		const entry = await cdn.get({ name });
```

Comment:
Useless const for single use. Ironic how you made a similar comment like this yesterday on schedule task. "Do as I say, not as I do" :p 

```suggestion
		const entry = await this.container.grpc.cdn.get({ name });
```